Gemfile

```diff
@@ -91,12 +65,14 @@ group :development, :test do
   gem "factory_girl_rails"
   gem "database_cleaner"
   gem "capybara-select2"
-  gem "show_me_the_cookies"
-  gem "brakeman"
+  gem 'brakeman', require: false
   gem "pry"
   gem "pry-remote"
   gem 'pry-byebug'
+  gem 'ruby-prof'
+end
+
+group :development do
```

Comment:
This is a valid point if we don't have access. But also I say that cause I notice on production we are using the default error pages (except for the 404 page) coming out of Rails instead of doing something custom? We don't even have a favicon or anything either in prod. Really doesn't matter to me honestly if we keep it in or not.